content/en/docs/Concepts/crds/bundleIndexImage.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Bundle Index Image"
+weight: 3
+description: >
+    Bundle index image example and references
```

Comment:
Description is supposed to describe WHAT exactly this document is going to enumerate. This just looks like the statement in `title` with just a couple of more words sprinkled after it.